Configurations/50-win-onecore.conf

```
@@ -61,4 +61,43 @@ my %targets = (
         ex_libs         => "onecore.lib",
         multilib        => "-arm64",
     },
+
+	# Universal Windows Platform (UWP) App Support
```

The format is not aligned

```suggestion
	# Universal Windows Platform (UWP) App Support
```